.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Ask a question
+    url: https://github.com/euphony-io/euphony/discussions
+    about: GitHub Discussions is a collaborative communication forum for the community around an open source project. Community members can ask and answer questions, share updates, have open-ended conversations, and follow along on decisions affecting the community's way of working.
```

Comment:
```suggestion
    about: Ask questions and discuss with other Euphony community members
```

Reply:
Sorry for checking late!

I saw your result of `issue template chooser` on your repo.
You did it. and I suggests some message modification! because this is too long description.

Let me approve it after this! Thanks 👍

Reply:
Thank you🙌